panic when posting telemetry.

<!-- If you suspect this could be a bug, follow the template. -->

### What version of Dgraph are you using?
v2.0.0-rc1

### Have you tried reproducing the issue with the latest release?
No.

### What is the hardware spec (RAM, OS)?
N/A

### Steps to reproduce the issue (command/config used to run Dgraph).

Not sure what steps there are to reproduce. Possibly network connection loss? I was running Dgraph on my laptop and noticed this the next day after opening it.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x150e4aa]

goroutine 230 [running]:
github.com/dgraph-io/dgraph/telemetry.NewAlpha(...)
	/tmp/go/src/github.com/dgraph-io/dgraph/telemetry/telemetry.go:81
github.com/dgraph-io/dgraph/edgraph.PeriodicallyPostTelemetry()
	/tmp/go/src/github.com/dgraph-io/dgraph/edgraph/server.go:108 +0x1ea
created by github.com/dgraph-io/dgraph/dgraph/cmd/alpha.setupServer
	/tmp/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/alpha/run.go:487 +0xb68
```

--- a/telemetry/telemetry.go
+++ b/telemetry/telemetry.go
@@ -51,12 +51,12 @@ const url = "https://ping.dgraph.io/3.0/projects/5b809dfac9e77c0001783ad0/events
 
 // NewZero returns a Telemetry struct that holds information about the state of zero server.
 func NewZero(ms *pb.MembershipState) *Telemetry {
-	if len(ms.Cid) == 0 {
+	if len(ms.GetCid()) == 0 {
 		glog.V(2).Infoln("No CID found yet")
 		return nil
 	}
 	t := &Telemetry{
-		Cid:       ms.Cid,
+		Cid:       ms.GetCid(),
 		NumGroups: len(ms.GetGroups()),
 		NumZeros:  len(ms.GetZeros()),
 		Version:   x.Version(),
@@ -78,7 +78,7 @@ func NewZero(ms *pb.MembershipState) *Telemetry {
 // NewAlpha returns a Telemetry struct that holds information about the state of alpha server.
 func NewAlpha(ms *pb.MembershipState) *Telemetry {
 	return &Telemetry{
-		Cid:     ms.Cid,
+		Cid:     ms.GetCid(),
 		Version: x.Version(),
 		OS:      runtime.GOOS,
 		Arch:    runtime.GOARCH,

